pages/trait/_trait.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+    <div>
+        <el-tabs type="border-card" v-model="tabIndex">
+            <el-tab-pane :label="seasonInfo.title" disabled>
+
+            </el-tab-pane>
+            <el-tab-pane label="챔피언 목록">
+                <ul>
+                    <li v-for="(unit, u_idx) in $store.state.unitList" :key="unit.id" class="unit-list">
```

Comment:
Codacy has a fix for the issue: [Replace `·v-for="(unit,·u_idx)·in·$store.state.unitList"·:key="unit.id"·class="unit-list"` with `⏎························v-for="(unit,·u_idx)·in·$store.state.unitList"⏎························:key="unit.id"⏎························class="unit-list"⏎····················`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9690624)
```suggestion
                    <li
```